.github/workflows/nodejs.yml

```diff
@@ -25,3 +22,68 @@ jobs:
         yarn test
       env:
         CI: true
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        ref: fix/docker-build
+    - name: clear old build files
+      run: |
+        rm -rf build.zip build
+        yarn install
+    - name: Building
+      run: |
+        yarn build
+    - name: Bundling
+      run: |
+        mkdir build && mv .next build/ && mv dist/* build/ && mv package.json build/ && mv yarn.lock build/ && mv Dockerfile build/
+        zip -q -r build.zip build
+    - name: Upload build Artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: build
+        path: ./build.zip
+    - name: Upload deploy script as an artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: deploy
+        path: ./deploy_script.sh
+    - name: Return to original state
+      run: |
+        git reset .
+        git checkout .
+        rm -rf build
+        echo "DONE!"
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/download-artifact@v1
+      with:
+        name: deploy
+        path: ./deploy_script.sh
+    - name: download build artifact
+      uses: actions/download-artifact@v1
+      with:
+        name: build
+        path: ./build.zip
+    - name: copy deploy_file via ssh
+      uses: appleboy/scp-action@master
+      env:
+        HOST: ${{ secrets.HOST }}
+        USERNAME: ${{ secrets.USERNAME }}
+        PORT: ${{ secrets.PORT }}
+        KEY: ${{ secrets.KEY }}
+      with:
+        source: "./deploy_script.sh,./build.zip"
+        target: "deploy_spotlights"
+    - name: executing remote ssh commands using ssh
+      uses: appleboy/ssh-action@master
+      with:
+        host: ${{ secrets.HOST }}
+        username: ${{ secrets.USERNAME }}
+        key: ${{ secrets.KEY }}
+        port: ${{ secrets.PORT }}
+        script: |
+          cd ./deploy_spotlights/deploy_script.sh && chmod +x deploy_script.sh && bash deploy_script.sh
```

Comment:
New line at end of file ... this should help https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig ... config file already exists on the repo.